pandas/_libs/tslibs/c_timestamp.pyx

```diff
@@ -381,5 +379,5 @@ cdef class _Timestamp(datetime):
 
     def timestamp(self):
         """Return POSIX timestamp as float."""
-        # py27 compat, see GH#17329
+        # Note: tz-aware Timestamps will return a UTC-like timestamp
```

Comment:
I think keeping the issue reference would be helpful.  It is for naive timestamp that we don’t match the stdlib.